accounts/accounts/controllers/authentication.py

```diff
@@ -93,6 +93,11 @@ def login(method: str, form_data: MultiDict, ip: str,
         logger.debug('Authentication failed for %s: %s', form.username.data, e)
         data.update({'error': 'Invalid username or password.'})
         return data, status.HTTP_400_BAD_REQUEST, {}
+    except Exception as e:
```

Comment:
Usually we wouldn't catch Exception but here we do, this is due to the critical nature of /login and the desire to not have a different response to the client on an unhandled exception.

Reply:
This makes sense. Is there evidence that this has been happening?

Reply:
I'm seeing 

`File "/users/e-prints/virtualenvs/arxiv-auth-bVddIfrH/lib/python3.6/site-packages/arxiv/users/legacy/authenticate.py", line 71, in authenticate
raise Unavailable('Database is temporarily unavailable') from e
arxiv.users.legacy.exceptions.Unavailable: Database is temporarily unavailable`

In the logs. I'll add a test for that.